Apps/Src/SendTritium.c

```diff
@@ -0,0 +1,442 @@
+/**
+ * @copyright Copyright (c) 2022 UT Longhorn Racing Solar
+ * 
+ * @file SendTritium.c
+ * @brief Function implementations for the SendTritium application.
+ * 
+ * This contains functions relevant to updating the velocity and current setpoitns
+ * of the Tritium motor controller. The implementation includes a normal current
+ * controlled mode, a one-pedal driving mode, cruise control, and regenerative braking.
+ * The logic is determined through a finite state machine implementation.
+ * 
+ * @author Nathaniel Delgado (NathanielDelgado)
+ * @author Diya Rajon (diyarajon)
+ * @author Ishan Deshpande (IshDeshpa)
+ */
+
+#include "Pedals.h"
+#include "ReadCarCAN.h"
+#include "Minions.h"
+#include "SendTritium.h"
+#include "ReadTritium.h"
+#include "CANbus.h"
+#include "UpdateDisplay.h"
+
+// Macros
+#define MAX_VELOCITY 50.0f // m/s    if rmp: 20000.0f // rpm
+#define NORMAL_BRAKE_THRESHOLD 5  // percent
+#define ONEPEDAL_BRAKE_THRESHOLD 20  // percent
+#define NEUTRAL_THRESHOLD 25    // percent
+#define DEBOUNCE_PERIOD 2 // in 100 ms
+#define VELOCITY_THRESHOLD 20.0f // m/s
+
+#define MOTOR_MSG_PERIOD 100
+#define FSM_PERIOD 20
+#define MOTOR_MSG_COUNTER_THRESHOLD (MOTOR_MSG_PERIOD)/(FSM_PERIOD)
+
+// Inputs
+static bool cruiseEnable = false;
+static bool cruiseSet = false;
+static bool regenToggle = false;
+static uint8_t brakePedalPercent = 0;
+static uint8_t accelPedalPercent = 0;
+
+static bool forwardGear = false;
+static bool neutralGear = false;
+static bool reverseGear = false;
+
+// Outputs
+static float currentSetpoint = 0;
+static float velocitySetpoint = 0;
+static float cruiseVelSetpoint = 0;
+static bool brakelightState = false;
+
+// Current observed velocity
+static float velocityObserved = 0;
```

Comment:
Do you use this anywhere? If it's only ever used in a testing context, it should probably be wrapped with the same macro guard as everything else. It could also be defined somewhere else and accessed, but that may be unnecessary overhead.